shim/README.md

```diff
@@ -0,0 +1,12 @@
+# `cc-shim`
+
+`cc-shim` is a process spawned by the runtime per container.The runtime 
+provides the pid of the cc-shim process to containerd-shim on OCI create command.
+
+`cc-shim` forwards all signals to the cc-proxy process to be handled by the agent
+in the VM.
+
+TODO:
+It should also forward any input received from containerd-shim to cc-proxy and 
+forward the stdout/stderr received from the proxy to containerd-shim.
```

Comment:
so cc-shim will own and manage of init-std* and exec named pipes provided by docker?